jumpscale/sals/process/__init__.py

```diff
@@ -49,26 +43,23 @@ def execute(
     replace_env=False,
     die=False,
 ):
-    """
-    execute a command.
-
-    accepts command as a list too, with auto-escaping.
+    """Execute a command.
 
-    Arguments:
-        cmd (str or list): command to be executed, e.g. `"ls -la"` or `["ls", "-la"]
+    Accepts command as a list too, with auto-escaping.
 
-    Keyword Arguments:
-        showout (bool): show stdout of the command (default: False)
-        cwd (str): specify a working directory for the command (default: None)
-        shell (str): specify a shell to execute the command (default: "/bin/bash")
-        timeout (int): timeout before kill the process (default: 600)
-        asynchronous (bool): execute in asynchronous mode or not (default: False)
-        env (dict): add environment variables here (default: {})
-        replace_env (bool): replace entire environment with env (default: False)
-        die (bool): die if command failed (default: False)
+    Args:
+        cmd (str/list[str]): Command to be executed, e.g. "ls -la" or ["ls", "-la"]
```

Comment:
```suggestion
        cmd (str or list of str): Command to be executed, e.g. "ls -la" or ["ls", "-la"]
```

Reply:
If we can follow this style (it's not strict at all, just for consistency)

Reply:
> If we can follow this style (it's not strict at all, just for consistency)

done module wide. thanks.